src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -122,6 +122,23 @@ public AcquireTokenForClientParameterBuilder WithPreferredAzureRegion(bool useAz
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// </summary>
+        public AcquireTokenForClientParameterBuilder WithFmiPath(string pathSuffix)
```

Comment:
This is only for use by MISE or is it also something for third partys?

Reply:
It will be used by 3p.

Reply:
Let's keep the summary vague for now, we're not ready to publicise this too broadly.